client/http/http.go

```diff
@@ -7,31 +7,64 @@ import (
 	"context"
 	"io"
 	"net/http"
+	"strconv"
 	"time"
 
+	"github.com/opentracing-contrib/go-stdlib/nethttp"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"github.com/prometheus/client_golang/prometheus"
+
 	"github.com/beatlabs/patron/correlation"
 	"github.com/beatlabs/patron/encoding"
 	"github.com/beatlabs/patron/log"
 	"github.com/beatlabs/patron/reliability/circuitbreaker"
 	"github.com/beatlabs/patron/trace"
-	"github.com/opentracing-contrib/go-stdlib/nethttp"
-	opentracing "github.com/opentracing/opentracing-go"
-	"github.com/opentracing/opentracing-go/ext"
 )
 
 const (
 	clientComponent = "http-client"
 )
 
+var (
+	reqTotalMetric   *prometheus.CounterVec
+	reqLatencyMetric *prometheus.HistogramVec
+)
+
+func init() {
+	reqTotalMetric = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "client",
+			Subsystem: "http",
+			Name:      "requests_total",
+			Help:      "Total number of HTTP requests sent by the client.",
+		},
+		[]string{"method", "url", "status_code"},
+	)
+	prometheus.MustRegister(reqTotalMetric)
+
+	reqLatencyMetric = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "client",
+			Subsystem: "http",
+			Name:      "requests_latency",
+			Help:      "Latency of a completed HTTP requests by the client.",
+		},
+		[]string{"method", "url", "status_code"},
+	)
+	prometheus.MustRegister(reqLatencyMetric)
+}
+
 // Client interface of a HTTP client.
 type Client interface {
 	Do(ctx context.Context, req *http.Request) (*http.Response, error)
 }
 
 // TracedClient defines a HTTP client with tracing integrated.
 type TracedClient struct {
-	cl *http.Client
-	cb *circuitbreaker.CircuitBreaker
+	cl             *http.Client
+	cb             *circuitbreaker.CircuitBreaker
+	metricsEnabled bool
```

Comment:
☝🏻  Let me know if `me` would be preferable

Reply:
Hey, I believe we should always enable them, so let's remove the field completely.